packages/nightingale-linegraph-track/src/nightingale-linegraph-track.js

```diff
@@ -59,8 +59,8 @@ class NightingaleLinegraphTrack extends ProtvistaTrack {
         d.colour = d.colour || interpolateRainbow(Math.random()); // eslint-disable-line no-param-reassign
         return this.drawLine(d)(d.values);
       })
-      .attr("fill", "none")
-      .attr("stroke", (d) => d.colour)
+      .attr("fill", (d) => d.area ? d.colour : "none")
```

Comment:
Could there be instances where we want a different stroke and fill colour? In that case maybe we should add a `fillColor` attribute to the configuration object, and this could be the flag to use instead of `area`?

Reply:
Good thinking. And is it `fillColor` or `fillColour` ? ;) I would personally change it to `color` and `fillColor`, and keep the `colour` back-compatible.

Reply:
👍  It should be `fillColor` - we're trying to stick to US spelling although a codebase-wide search/replace might be needed at some point 😄

Reply:
ok so it will be `fillColor` and `colour` for now?

Reply:
Sounds good, we'll update later